### PR TITLE
Allow attaching up to 5 EBS volumes.

### PR DESCRIPTION
**Pull request by rxin**

### CODE REVIEW

File: setup-slave.sh [hunk 1]
```diff
@@ -40,30 +40,43 @@ fi
...
-# Format and mount EBS volume (/dev/sdv) as /vol if the device exists
-if [[ -e /dev/sdv ]]; then
-  # Check if /dev/sdv is already formatted
-  if ! blkid /dev/sdv; then
-    mkdir /vol
-    yum install -q -y xfsprogs
-    if mkfs.xfs -q /dev/sdv; then
-      mount -o $XFS_MOUNT_OPTS /dev/sdv /vol
-      chmod -R a+w /vol
+function setup_ebs_volumn {
```
Comment (shivaram) [inline setup-slave.sh]: typo volumn -> volume ?

File: setup-slave.sh [hunk 2]
```diff
@@ -40,30 +40,43 @@ fi
...
+
+# Format and mount EBS volume (/dev/sd[v, w, x, y, z]) as /vol[x] if the device exists
+setup_ebs_volumn /dev/sdv /vol0
+setup_ebs_volumn /dev/sdw /vol1
+setup_ebs_volumn /dev/sdx /vol2
+setup_ebs_volumn /dev/sdy /vol3
+setup_ebs_volumn /dev/sdz /vol4
+
+# Alias vol to vol0 for backward compatibility
+ln -s /vol /vol0
```
Comment (shivaram) [inline setup-slave.sh]: You should do this only if /vol0 exists. (See that we do rmdir $mount_point in case of mkfs error)

**Comment by rxin** [comment]
Pushed a new version.

**Comment by shivaram** [comment]
LGTM. Just to confirm, you launched a cluster with this ?

**Comment by rxin** [comment]
yup

**Pull request merged**